packages/rum-core/src/browser/htmlDomUtils.ts

```diff
@@ -43,3 +43,13 @@ export function forEachChildNodes(node: Node, callback: (child: Node) => void) {
 export function getParentNode(node: Node): Node | null {
   return isNodeShadowRoot(node) ? node.host : node.parentNode
 }
+
+export function getClassList(element: Element): DOMTokenList | string[] {
+  // ie 11 does not support classList on SVG and MathMl elements
+  if (element.classList) {
+    return element.classList
+  }
+
+  // Use element.getAttribute('class') because element.className on SVG elements is a SVGAnimatedString
```

Comment:
💭 **Thought**: `classList` is not supported on `MathML` elements as well. It would make sense to reflect it in the comment (if the `MathML` case is covered)

Reply:
It's true that my comments were confusing. I tried to improve them. Let me know what you think